besu/src/test/java/org/hyperledger/besu/cli/BesuCommandTest.java

```diff
@@ -805,7 +805,7 @@ public void configOptionDisabledUnderDocker() {
 
   @Test
   public void nodekeyOptionMustBeUsed() throws Exception {
-    final File file = new File("./specific/key");
+    final File file = new File("./specific/enclavePrivateKey");
```

Comment:
I believe `key` was replaced by `enclavePrivateKey` by mistake (this has happened all over this class and maybe in other places as well). Likely to be an IDE refactor doing too much :)

Reply:
Nope I did this on purpose :)